server/rest/src/test/java/org/infinispan/rest/resources/ContainerResourceTest.java

```diff
@@ -205,6 +214,7 @@ public void testCaches() {
 
       List<String> bounded = find(jsonNode, "bounded");
       assertTrue(bounded.contains("false"));
+      assertTrue(bounded.contains("true"));
```

Comment:
yes, because one of the caches is bounded but not the rest

Reply:
I understand that, I was asking if there was a way to make it more strict. TBH even just changing the assert so that bounded is true once and rest are all false would make me feel a bit better.